commit 0766a3ca5473a9e126e91022075b4b3798b8d5bc
Author: Arkadiusz Bubala <arkadiusz.bubala@open-e.com>
Date:   Thu Aug 6 08:56:59 2015 +0200

    Increase the length of description field.
    
    Make description field larger to satisfy all possible pids and file
    descriptor values.

diff --git a/lib/ipc_int.h b/lib/ipc_int.h
index a428721..500315e 100644
--- a/lib/ipc_int.h
+++ b/lib/ipc_int.h
@@ -159,7 +159,7 @@ enum qb_ipcs_connection_state {
 	QB_IPCS_CONNECTION_SHUTTING_DOWN,
 };
 
-#define CONNECTION_DESCRIPTION (16)
+#define CONNECTION_DESCRIPTION (34) /* INT_MAX length + 3 */
 
 struct qb_ipcs_connection_auth {
 	uid_t uid;
